packages/changelog-linker/tests/ChangelogDumper/WithTagsTest.php

```diff
@@ -36,8 +36,7 @@ public function testReportChanges(): void
 
         $content = $this->changelogDumper->reportChangesWithHeadlines($this->changes, false, false, 'categories');
 
-        $expectedFile = __DIR__ . '/WithTagsSource/expected1.md';
-        $this->assertStringEqualsFile($expectedFile, $content);
+        $this->assertStringEqualsFile(__DIR__ . '/WithTagsSource/expected1.md', $content);
```

Comment:
:+1:

Reply:
this actually fine if variable will be used in next statement, reverted.